src/OpenLoco/Windows/TitleLogo.cpp

```diff
@@ -41,7 +41,7 @@ namespace OpenLoco::Ui::Windows
             &_events);
 
         window->widgets = _widgets;
-        window->enabled_widgets = 1 << Widx::logo;
+        window->setVisible(Widx::logo);
```

Comment:
This is different. setVisible appends and doesn't assign. Need to think of a better way.